.changesets/add-sample-data-and-error-helpers.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "add"
+---
+
+Add sample data and error helpers.
```

Comment:
Some usage examples (that we can later copy-paste) in the docs would be helpful. See PR #124 for examples.